packages/payload/src/admin/components/views/collections/List/Cell/index.tsx

```diff
@@ -52,6 +53,19 @@ const DefaultCell: React.FC<Props> = (props) => {
     }
   }
 
+  if (field.name === 'id' && cellData.toString().length < 5) {
```

Comment:
Hmmm should just use `CodeCell` no matter the length? This will provide the most consistency.

Reply:
@jacobsfletch This has been resolved.